script/Ciel/Day 13/13_01_01_CIEL13_2.txt

```diff
@@ -362,22 +384,24 @@ I'm so glad that she's still alive―――
 [sha:fbb155b38d0ec30cd7384c989e8cd95d5245b384]{
 -- Page 93, Offset 33415.
 -- ソレは折れた骨のまま、ガクン、と顔をあげた。
-With her bones still broken, she swiftly raises her head.
+Its head snaps up, broken bones grinding in protest.
 }
 [sha:1835faa52a5ab3af3d25e419eafc808747eb64e9]{
 -- Page 93, Offset 33416.
 -- 青白い肌。生気の欠けた<眼|まなこ>。呼吸をしない口。
+// Kohaku jumpscare
 Pale skin. Lifeless eyes. An unbreathing mouth.
```

Comment:
Despite it not being entirely grammatical I think you can just drop the "an" for the mouth part. Would rather have it fit together like that than anything else.